### ISSUE TITLE
trouble proving derive PartialEq with v0.1

### ISSUE THREAD

**Issue by Emilie-Thome** [creusot-rs/creusot#1023]
Unless I did something wrong, I can not prove `#[derive(creusot_contracts::prelude::PartialEq)]` with why3 and Creusot v0.1.
Here is an extract of something I want proved:

```
use creusot_contracts::{ensures, logic, open, prelude, requires, DeepModel};
#[derive(prelude :: Clone, Copy, prelude :: PartialEq, prelude :: Default, DeepModel)]
pub enum Braking {
    #[default]
    NoBrake,
    SoftBrake,
    UrgentBrake,
}
```

To my knowledge, only the `PartialEq` is not proved by why3.

**Comment by xldenis** [comment]
Hmm, that's weird, I'm not able to reproduce the issue, can you provide exactly the commands that you used to create the files ?

**Comment by xldenis** [comment]
That's the whole file?

**Comment by xldenis** [comment]
Aha, this is an occurence of #1014. We need to backport this patch and push a `v0.1.1`

**Comment by xldenis** [comment]
Fixed in `0.1.1`
